### PR TITLE
Added tmux and bash-completion

### PR DESCRIPTION
**Pull request by cavenditti**
Just added the packages for convenience when testing

### CODE REVIEW

File: pkgs/bash-completion/build.textproto [hunk 1]
```diff
@@ -0,0 +1,9 @@
+source: "https://github.com/scop/bash-completion/archive/2.9.tar.gz"
+hash: "fddd9e6739c972eea95d1f0e0e84fa9a17a5935ba21eab37308b27580cc4d790"
+version: "2.9-1"
+
+cbuilder: <
+  autoreconf: true
+  copy_to_builddir: true
+>
+
```
Comment (stapelberg) [inline pkgs/bash-completion/build.textproto]: Please remove this empty line.

File: pkgs/libevent/build.textproto [hunk 2]
```diff
@@ -0,0 +1,7 @@
+source: "https://github.com/libevent/libevent/releases/download/release-2.1.11-stable/libevent-2.1.11-stable.tar.gz"
+hash: "a65bac6202ea8c5609fd5c7e480e6d25de467ea1917c08290c521752f147283d"
+version: "2.1.11"
```
Comment (stapelberg) [inline pkgs/libevent/build.textproto]: Please make this 2.1.11-1

File: pkgs/tmux/build.textproto [hunk 3]
```diff
@@ -0,0 +1,11 @@
+source: "https://github.com/tmux/tmux/releases/download/2.9/tmux-2.9.tar.gz"
+hash: "34901232f486fd99f3a39e864575e658b5d49f43289ccc6ee57c365f2e2c2980"
+version: "2.9"
```
Comment (stapelberg) [inline pkgs/tmux/build.textproto]: Please make this 2.9-1

File: pkgs/tmux/build.textproto [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+cbuilder: <
+  copy_to_builddir: true
+>
+
+dep: "libevent"
+dep: "ncurses"
+
```
Comment (stapelberg) [inline pkgs/tmux/build.textproto]: Please remove this empty line.

File: pkgs/tmux/build.textproto [hunk 5]
```diff
@@ -0,0 +1,11 @@
+source: "https://github.com/tmux/tmux/releases/download/2.9/tmux-2.9.tar.gz"
+hash: "34901232f486fd99f3a39e864575e658b5d49f43289ccc6ee57c365f2e2c2980"
+version: "2.9"
+
+cbuilder: <
+  copy_to_builddir: true
```
Comment (stapelberg) [inline pkgs/tmux/build.textproto]: Just checking: does tmux (and libevent) not build without copy_to_builddir?

File: pkgs/libevent/build.textproto (hunk 2 above)
Comment (cavenditti) [reply pkgs/libevent/build.textproto]: It took me a while to realize why they were this way, this was the very first thing I've done and I didn't know about the scaffold command yet

File: pkgs/tmux/build.textproto (hunk 5 above)
Comment (cavenditti) [reply pkgs/tmux/build.textproto]: Yes, they both fail without copy_to_builddir

**Comment by stapelberg** [comment]
Thanks!

**Pull request merged**